linode/manager
# style: [M3-6639] - Add outline to country flags that contain white

## Description 📝
- Adds an outline to some country flags so the contrast in light mode is better

## Major Changes 🔄
- Added outline to `jp`, `sg`, and `id` flags
- Added Storybook story for the `<Flag />` component 

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2023-06-21 at 11 05 23 AM](https://github.com/linode/manager/assets/115251059/e90b00c3-3925-4ed9-a1d4-c46eea24d96d) | ![Screenshot 2023-06-21 at 11 04 49 AM](https://github.com/linode/manager/assets/115251059/d6eccb53-6cd2-4270-bfa1-057fb96c9b11) |

## New Storybook Story 🤤
![Screenshot 2023-06-21 at 11 09 33 AM](https://github.com/linode/manager/assets/115251059/b7e3f80a-ea10-4934-b829-235c3d71410c)

## How to test 🧪
- Pull the PR
- Switch to **Light Mode**
- Go to the Linode Create page
- Open the Region Select component
- Verify that there is a subtitle outline around the `jp`, `sg`, and `id` flags
- Optionally, open storybook and verify you see the outline there too
